tools/validate_paths.go

```diff
@@ -230,14 +248,41 @@ func checkFiles(knownOC map[string]pathType, files []string) ([]file, error) {
 			}
 		}
 
-		if len(lines) == 0 && len(errs) == 0 {
+		if len(lines) == 0 && len(errs) == 0 && len(dependencies) == 0 {
 			continue
 		}
-		report = append(report, file{name: f, lines: lines, errors: errs})
+		report = append(report, file{name: f, lines: lines, dependencies: dependencies, errors: errs})
 	}
+	report = validateDependency(validProfile, report)
 	return report, nil
 }
 
+// getFeatureProfileNameFromPath gets feature profile id.name from path
+func getFeatureProfileNameFromPath(file string, fp fppb.FeatureProfile) string {
+	featureProfileFilePath := strings.Split(strings.SplitAfter(file, featuresRoot)[1], "/")
+	featureProfileFilePath = featureProfileFilePath[1 : len(featureProfileFilePath)-1]
```

Comment:
I think this should be `[0:len(featureProfileFilePath)-1]`. validate_paths includes the `/feature` part